src/components/common/target-list-item.scss

```diff
@@ -1,41 +1,54 @@
 @import '../../style/common/constants';
 
 .target-item {
-  width: 25vw;
+  width: 26vw;
   display: flex;
-  padding-right: 0;
-  margin-right: 0;
-  margin-left: 2em;
-  padding-bottom: 0;
 }
 
 .target-item:hover {
   border: 4px solid $dodger-blue;
 }
 
+.target-item__trash-button-container {
+  text-align: left;
+  padding-top: 0.2em;
+  height: 2em;
+}
+
+.target-item__trash-button-img {
+  width: 30px;
+}
+
+.target-item__trash-button:hover {
+  background-color: $alizarin-crimson;
+}
+
 .target-item__text-container {
-  align-self: left;
+  text-align: left;
   flex-grow: 2;
+  margin-left: 2vw;
+  padding-left: 1em;
 }
+
 .target-item__title {
   font-family: $font-family-bold;
   font-size: 1em;
   line-height: 2em;
-  text-align: left;
   padding-left: 1vw;
-  padding-top: 0;
-  margin-top: 0;
+  padding-top: 0.2em;
+  padding-bottom: 0.8em;
```

Comment:
maybe use a shorcut property for setting padding here: 
padding: .2em 0 0.8em 1vw
it follows this convention --> padding: padding-top padding-right padding-bottom padding-left

Also try to use the same unit to set different values of the same property